resources/views/layouts/partials/header/autocomplete.blade.php

```diff
@@ -50,5 +44,19 @@ class="contents"
                 </div>
             </div>
         </ais-instant-search>
+        <div v-else class="relative w-full">
+            {{-- TODO: Do we still need this double input? --}}
+            <form name="autocomplete-form" id="autocomplete-form" method="get" action="{{ route('search') }}" class="flex flex-row relative">
+                <x-rapidez::input
+                    type="search"
+                    name="q"
+                    :placeholder="__('What are you looking for?')"
+                    v-model="$root.autocompleteFacadeQuery"
```

Comment:
Can you explain this one?

Reply:
https://github.com/rapidez/core/pull/768#discussion_r2001072653
> It maintains your current input in the autocompleteFacadeQuery 
If you start typing before Vue has even booted, it will simply catch up due to that v-model.
>
>Once it has downloaded and prepared the real autocomplete it will replace this loading component, refine using autocompleteFacadeQuery thus setting the new input to the current search query